src/xbps/updating.md

```diff
@@ -1,47 +1,37 @@
 # Updating
 
-Like any other system it is important to keep Void Linux up to date.
-
-In general Void should be updated with an XBPS invocation:
+Like any other system, it is important to keep Void up-to-date. Use
+[xbps-install(1)](https://man.voidlinux.org/xbps-install.1) to update:
 
 ```
 # xbps-install -Su
 ```
 
-> Note: XBPS must use a separate transaction to update itself. If your first
-> update includes the package `xbps`, you will need to run an additional update
-> for the rest of the system.
+Note: XBPS must use a separate transaction to update itself. If your update
+includes the `xbps` package, you will need to run the above command a second
+time to apply the rest of the updates.
```

Comment:
As I mentioned in #297, I believe this shouldn't be a note.

Reply:
Fixed.